docs/4-user-guide/1-the-zarf-cli/index.md

```diff
@@ -52,28 +52,33 @@ First, lets test to make sure the CLI you have works by running the CLI to get a
 The output of the help command should look <b>something</b> like this:
 
 ```text
-Small tool to bundle dependencies with K3s for air-gapped deployments
+Zarf eliminates the complexity of air gap software delivery for Kubernetes clusters and cloud native workloads
+using a declarative packaging strategy to support DevSecOps in offline and semi-connected environments.
 
 Usage:
   zarf [COMMAND]|[ZARF-PACKAGE]|[ZARF-YAML] [flags]
   zarf [command]
 
 Available Commands:
   completion        Generate the autocompletion script for the specified shell
-  connect           Access services or pods deployed in the cluster.
+  connect           Access services or pods deployed in the cluster
   destroy           Tear it all down, we'll miss you Zarf...
   help              Help about any command
   init              Prepares a k8s cluster for the deployment of Zarf packages
-  package           Pack and unpack updates for the Zarf gitops service.
+  package           Zarf package commands for creating, deploying, and inspecting packages
   prepare           Tools to help prepare assets for packaging
   tools             Collection of additional tools to make airgap easier
   version           Displays the version of the Zarf binary
 
 Flags:
   -a, --architecture string   Architecture for OCI images
   -h, --help                  help for zarf
-  -l, --log-level string      Log level when running Zarf. Valid options are: warn, info, debug, trace
-  -t, --toggle                Help message for toggle
+  -l, --log-level string      Log level when running Zarf. Valid options are: warn, info, debug, trace (default "info")
```

Comment:
Since flags will change more often than top-level commands, I think we can just delete the flags from this excerpt.

Reply:
Do you think it would be worth putting in a note somewhere saying that different flag options will be in the output too, just so it's mentioned?

Reply:
Yeah that's probably fine, just don't want the team to worry about checking this as we change flags over time as CI won't catch it.